frontend/src/components/scan/ScanForm.jsx

```diff
@@ -104,7 +106,7 @@ const observableType2PropsMap = {
 };
 const initialValues = {
   classification: "ip",
-  observable_name: "",
+  observable_names: [""],
```

Comment:
this is not empty.

Reply:
Assuming you were referring to error detection, does it work better now?